### PR TITLE
Release: Changelog 0.104.0

### PR DESCRIPTION
**Pull request by Jonathan-Rosenberg**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 ## UNRELEASED
 
+## v0.103.1
+
+:new: What's new:
+- Add relative ./ path support for MD images (#6057)
+- UI: Support uploading multiple files and directories (#6074)
+- UI: show summary of uploaded files (#6079)
```
Comment (ozkatz) [inline CHANGELOG.md:10]: this is a follow up to #6047, I think we can remove this line.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+
+:bug: Bugs fixed:
+- Docs: fix lakefs_object and invalid sql statements (#6072)
```
Comment (ozkatz) [inline CHANGELOG.md:13]: Remove. Docs are detached from version releases.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,20 @@
...
+- Delta Diff fails on certain operations (#6065)
+- Fix: URI parse with whitespace (#6080)
```
Comment (ozkatz) [inline CHANGELOG.md:15]: Might want to preface with the affected component.

```suggestion
- Fix (HadoopFS): URI parse with whitespace (#6080)
```

**Review by ozkatz — CHANGES_REQUESTED** [review]
Would remove the docs changes from the change log.

non-blocking: since there's new functionality - shouldn't we release a minor?

File: CHANGELOG.md (hunk 3 above)
Comment (Jonathan-Rosenberg) [reply CHANGELOG.md:15]: Thanks, another catch: HadoopFS is also detached from version releases...

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,20 @@
 
 ## UNRELEASED
 
+## v0.103.1
+
+:new: What's new:
+- Add relative ./ path support for MD images (#6057)
```
Comment (N-o-Z) [inline CHANGELOG.md:8]: This is a bug fix

**Review by N-o-Z — CHANGES_REQUESTED** [review]
One comment

**Review by ozkatz — APPROVED** [review]
LGTM!

**Review by N-o-Z — APPROVED** [review]

**Pull request merged**